cluster/mesos/docker/util.sh

```diff
@@ -288,6 +283,8 @@ function kube-up {
 
   echo "Starting ${KUBERNETES_PROVIDER} cluster" 1>&2
   cluster::mesos::docker::docker_compose up -d
+  echo "Scaling ${KUBERNETES_PROVIDER} cluster to ${NUM_MINIONS} slaves"
+  cluster::mesos::docker::docker_compose scale mesosslave=${NUM_MINIONS}
```

Comment:
bugs me that scale can't be done in the yaml, but it looks like that conversation has been going on for more than a year in the fig/compose repo.